wolfssl/wolfcrypt/port/st/stm32.h

```diff
@@ -186,6 +186,14 @@ int  wc_Stm32_Hash_Final(STM32_HASH_Context* stmCtx, word32 algo,
         #define STM_CRYPT_TYPE uint8_t
     #endif
 
+    /* newer crypt HAL requires auth header size as 4 bytes (word) */
```

Comment:
I think this is technically `STM32_HAL_V4` (at least for CRYP), but the versioning is weird, so I think stick with this :)